cmd/fireeth/poller.go

```diff
@@ -2,12 +2,13 @@ package main
 
 import (
 	"fmt"
-	"github.com/streamingfast/cli"
 	"path"
 	"strconv"
 	"strings"
 	"time"
 
+	"github.com/streamingfast/cli"
+
 	"github.com/spf13/cobra"
```

Comment:
```suggestion
	"github.com/streamingfast/cli"
    "github.com/spf13/cobra"
```

+ another round of `goimports`

Reply:
Thank you for your review. Modified.